<!-- -->
<launch>
    <param name="/oa/baseline" 				type="double" value="0.10" />
    <param name="/oa/downsample_scale" 			type="double" value="2.0" />
    <param name="/oa/low_occ_thresh"			type="double" value="0.9" />
    <param name="/oa/high_occ_thresh"			type="double" value="1.8" />
    <param name="/oa/sensor_pixel_error"		type="double" value="0.0" />

    <node pkg="disparity_expansion" type="disparity_expansion_clean" name="disparity_expansion">
        <param name="lut_max_disparity" type="int" value="100" />
        <param name="robot_radius" type="double" value="0.5" />
        <param name="bg_multiplier" type="double" value="2.0" />
        <param name="padding" type="double" value="-1" />
        <param name="sensor_pixel_error" type="double" value="0.2" />
        <!--param name="downsample_scale" type="double" value="4.0" /-->
	<remap from="/nerian_sp1/right/camera_info" to="/camera/depth/camera_info"/>
	<remap from="/nerian_sp1/disparity_map_32F" to="/camera/disparity_sensor_msg/image_raw"/>	
    </node>
    <!-- node pkg="disparity_expansion" type="disparity_pcd" name="disparity_pcd_real">
	<remap from="/nerian_sp1/left/image_raw" to="/camera/rgb/image_raw"/>	
	<remap from="/disparity" to="/camera/disparity_sensor_msg/image_raw"/>
	<remap from="/nerian_sp1/right/camera_info" to="/camera/depth/camera_info"/>
	<remap from="/pcd" to="/pcd_real"/>
        <param name="downsample_scale" type="double" value="1.0" />
    </node>
    <node pkg="disparity_expansion" type="disparity_pcd" name="disparity_pcd_fg">
	<remap from="/nerian_sp1/left/image_raw" to="/camera/rgb/image_raw"/>	
	<remap from="/disparity" to="/ceye/left/expanded_disparity_fg"/>
	<remap from="/pcd" to="/pcd_fg"/>
        <param name="downsample_scale" type="double" value="2.0" />
	<remap from="/nerian_sp1/right/camera_info" to="/camera/depth/camera_info"/>
    </node>
    <node pkg="disparity_expansion" type="disparity_pcd" name="disparity_pcd_bg">
	<remap from="/nerian_sp1/left/image_raw" to="/camera/rgb/image_raw"/>	
	<remap from="/disparity" to="/ceye/left/expanded_disparity_bg"/>
	<remap from="/pcd" to="/pcd_bg"/>
        <param name="downsample_scale" type="double" value="2.0" />
	<remap from="/nerian_sp1/right/camera_info" to="/camera/depth/camera_info"/>
    </node -->
</launch>
